src/Query_Oracle_Abstract.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+// vim:ts=4:sw=4:et:fdm=marker
+
+namespace atk4\dsql;
+
+/**
+ * Perform query operation on Oracle server.
+ *
+ * @license MIT
+ * @copyright Agile Toolkit (c) http://agiletoolkit.org/
+ */
+abstract class Query_Oracle_Abstract extends Query
+{
```

Comment:
please get rid of this class..

Reply:
leaving it for now because i have feeling that something could go there - Oracle specific feature which all Oracle versions share.